cloud/aws/go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/cdktf/cdktf-provider-aws-go/aws/v19 v19.54.0
 	github.com/cdktf/cdktf-provider-docker-go/docker/v11 v11.0.0
 	github.com/getkin/kin-openapi v0.113.0
-	github.com/golang/mock v1.6.0
+	github.com/golang/mock v1.7.0-rc.1
```

Comment:
Have we considered moving to the Uber fork https://github.com/uber-go/mock? The original repo is no longer maintained.